hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/JobMetrics.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.core;
+
+import com.hazelcast.spi.annotation.Beta;
+
+import javax.annotation.Nonnull;
+import java.util.Collection;
+
+/**
+ * Contains information related to job-specific metrics.
+ */
+@Beta
+public interface JobMetrics {
```

Comment:
do we need an interface here? it would just be a DTO.

Reply:
ok, will change